frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -509,6 +514,37 @@ export const WebGoalsTile = ({
         </div>
     )
 }
+
+export const WebExternalClicksTile = ({
+    query,
+    insightProps,
+}: {
+    query: DataTableNode
+    insightProps: InsightLogicProps
+}): JSX.Element | null => {
+    const { shouldStripQueryParams } = useValues(webAnalyticsLogic)
+    const { setShouldStripQueryParams } = useActions(webAnalyticsLogic)
+    return (
+        <div className="border rounded bg-bg-light flex-1 flex flex-col">
+            <div className="flex flex-row items-center justify-end m-2 mr-4">
+                <div className="flex flex-row items-center space-x-2">
+                    <LemonSwitch
+                        label={
+                            <div className="flex flex-row space-x-2">
```

Comment:
Do we need the wrapping div when this is just a single element? Perhaps this could all just be a string

Reply:
It could, doing this now!